nano/node/bootstrap/bootstrap_server.cpp

```diff
@@ -22,22 +22,17 @@ void nano::bootstrap_listener::start ()
 	listening_socket->start (ec);
 	if (ec)
 	{
-		node.logger.try_log (boost::str (boost::format ("Error while binding for incoming TCP/bootstrap on port %1%: %2%") % listening_socket->listening_port () % ec.message ()));
+		node.logger.always_log (boost::str (boost::format ("Network: Error while binding for incoming TCP/bootstrap on port %1%: %2%") % listening_socket->listening_port () % ec.message ()));
 		throw std::runtime_error (ec.message ());
 	}
 	debug_assert (node.network.endpoint ().port () == listening_socket->listening_port ());
 	listening_socket->on_connection ([this] (std::shared_ptr<nano::socket> const & new_connection, boost::system::error_code const & ec_a) {
-		bool keep_accepting = true;
 		if (ec_a)
 		{
-			keep_accepting = false;
-			this->node.logger.try_log (boost::str (boost::format ("Error while accepting incoming TCP/bootstrap connections: %1%") % ec_a.message ()));
+			return false;
```

Comment:
The callback should always return true (continue to accept).
There is no point stopping to accept accepts.
We have the half a second throttling to avoid fast loops.

Reply:
On reflection, it does make sense to keep the temporary error check.
We do not want to sleep for a half a second just because of EAGAIN.
I would ideally like all errors to be reported to the callback and the callback to decide if we should carry on or not but it is not abnormal to treat EAGAIN a special maskable error. So I suggest we keep the temporary errors check but only check for EAGAIN/EWOULDBLOCK.

Reply:
My thougts around this. I think there is a point. If it's not considered "temporary" (retry directly) or "system" (retry with some delay) (maybe some update of the terminology could make it clearer?) then it's a "permanent" error related to the socket/ip address changed etc. (e.g. EBADF, ENOTSOCK etc), which would for proper handling require tearing down the socket and recreating it. There is no such functionality from before, and I think we decided it is not in scope for this fix. So in those "permanent" error cases the option is to stop accepting (as was done before for ANY error), or keep accepting with no chance of it ever working again. Both will result in loss of ability to accept any more connections until the process is restarted. 

I feel both ways could work for this "hotfix", and we should implement the one with least amount of changed code/flows needed.